learn/work/fairness-readmission/index.qmd

```diff
@@ -0,0 +1,516 @@
+:::
+
+<!-- TODO: remove the /dev from the above URL -->
+
+### Evaluation
+
+We can now evaluate the workflows we've defined against resamples using our metric set. The `workflow_map()` function will call `tune_grid()` on each workflow:
+
+```{r}
+#| eval: !expr eval_fits
+#| label: "wflow-map"
+set.seed(1)
+wflow_set_fit <- 
+  workflow_map(
+    wflow_set, 
+    verbose = TRUE, 
+    seed = 1, 
+    metrics = m_set,
+    resamples = readmission_folds
+  )
+```
+
+```{r}
+#| label: "save-wflow-set"
+#| eval: !expr eval_fits
+#| include: false
+save(wflow_set_fit, file = saved_fits)
+```
+
+A fitted workflow set looks just like the unfitted workflow set we saw previously, except that information on the tuning process is now stored in the `option` and `result` variables for each modeling workflow:
+
+```{r}
+#| label: "print-wflow-set"
+wflow_set_fit
+```
+
+## Model Selection
+
+Now that we've evaluated a number of models with a variety of metrics, we can explore the results to determine our optimal model. Beginning by a quick exploratory plot of the distributions of our metrics:
+
+```{r}
+#| label: "plot-wflow-set-fit-corr"
+#| message: false
+#| warning: false
+#| out.width: 80%
+#| fig.width: 7
+#| fig.height: 7
+wflow_set_fit %>%
+  collect_metrics() %>%
+  pivot_wider(
+    id_cols = c(wflow_id, .config), 
+    names_from = .metric, 
+    values_from = mean
+  ) %>%
+  select(-c(wflow_id, .config)) %>%
+  ggpairs() +
+  theme(axis.text.x = element_text(angle = 45, vjust = .8, hjust = .9))
+```
+
+The fairness metrics `demographic_parity()`, `equal_opportunity()`, and `equalized_odds()` all take values very close to zero for many models. Also, the metric values are highly correlated with each other, including correlations between fairness metrics and the more general-purpose performance metrics. That is, the most performant models also seem to be among the most fair.
+
+More concretely, we can rank the model configurations to examine only the most performant models:
+
+```{r}
+#| label: "rank-results"
+rank_results(wflow_set_fit, rank_metric = "roc_auc") %>%
+  filter(.metric == "roc_auc")
+```
+
+Almost all of the most performant model configurations arise from the boosted tree modeling workflow. Let's examine the results for specifically the modeling workflow that encodes age as a number more thoroughly:
+
+```{r}
+#| label: "autoplot-age-bt"
+#| out.width: 70%
+#| fig.width: 6
+#| fig.height: 7
+autoplot(wflow_set_fit, id = "age_bt")
+```
+
+The learning rate `learn_rate` seems to have a more pronounced effect on the results metrics than the number of randomly selected predictors `mtry`. As before, we see that the most performant models with respect to `roc_auc()` tend to be the most fair with respect to our fairness metrics. Further, the values of each of the fairness metrics plotted above seem highly correlated. 
```

Comment:
Minor, is it confusing that the words "with respect to" are used quite close to eachother?

Reply:
[e21af27](https://github.com/tidymodels/tidymodels.org/pull/52/commits/e21af27668c751d5159b7f0539c321000860814f).